README.md

```diff
@@ -35,12 +35,26 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
 
 1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
 2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
+
+If you want to install the dashboard into tekton-pipelines namespace:
 3. Install deployment config `oc process -f config/templates/deploy.yaml | oc apply -f -`
 4. Install build config `oc process -f config/templates/build.yaml | oc apply -f -`
 5. Wait until the pod `tekton-dashboard-1` is running in the tekton-pipelines namespace
 
+If you want to install the dashboard into any other namespace:
+3. Change line 47 in config/templates/deploy.yaml to `namespace: INSTALL_NAMESPACE`
+4. Run the command `chmod +x ./config/templates/minishiftInstallDashboard.sh`
+5. Install the Dashboard into the namespace desired `./config/templates/minishiftInstallDashboard.sh -n NAMESPACE`
+6. Wait until the pod `tekton-dashboard-1` is running in the tekton-pipelines namespace
+
+To uninstall the dashboard if created using the `minishiftInstall.sh` script:
+1. Run the command `chmod +x ./config/templates/minishiftDeleteDashboard.sh` 
```

Comment:
Same comments as per the install around committing it chmodded with a more appropriate file name, in a better place